crates/dslab-mp/src/mc/predicates.rs

```diff
@@ -167,6 +179,21 @@ pub mod goals {
             }
         })
     }
+
+    /// Checks if trace to given state has at least `n` events matching the predicate.
+    pub fn event_happened_n_times<F>(predicate: F, n: usize) -> GoalFn
```

Comment:
Предлагаю уточнить что это текущий run

```suggestion
    /// Checks if current run trace has at least `n` events matching the predicate.
    pub fn event_happened_n_times_current_run<F>(predicate: F, n: usize) -> GoalFn
```